File: assignment17.py
from numpy import var


vars = [0, 1]

def functionIntegerChecker(list_of_nums):
    has_seven_or_four = False
    for i in list_of_nums:
        if i == 7 or i == 4:
            has_seven_or_four = True
    
    if has_seven_or_four == True:
        print('the list has a seven and/or four')
    else:
        print('the list does not have a seven or four')

functionIntegerChecker(vars)

File: javascript_assignments/assignment17.js
let vars = [0, 1]

function functionIntegerChecker(list_of_nums){
    let has_seven_or_four = false
    for (let i = 0; i < list_of_nums.length; i++){
        if (i == 7 || i == 4 ){
            has_seven_or_four = true
        }
    }
    if (has_seven_or_four == true){
        console.log("the list has a seven and/or four")
    } else {
        console.log("the list does not have a seven or four")
    }
}

functionIntegerChecker(vars)